Sources/XcodeProj/Scheme/XCScheme+LaunchAction.swift

```diff
@@ -161,6 +162,93 @@ public extension XCScheme {
             super.init(preActions, postActions)
         }
 
+        public convenience init(
+            runnable: Runnable?,
+            buildConfiguration: String,
+            preActions: [ExecutionAction] = [],
+            postActions: [ExecutionAction] = [],
+            macroExpansion: BuildableReference? = nil,
+            selectedDebuggerIdentifier: String = XCScheme.defaultDebugger,
+            selectedLauncherIdentifier: String = XCScheme.defaultLauncher,
+            launchStyle: Style = .auto,
+            askForAppToLaunch: Bool? = nil,
+            pathRunnable: PathRunnable? = nil,
+            customWorkingDirectory: String? = nil,
+            useCustomWorkingDirectory: Bool = false,
+            ignoresPersistentStateOnLaunch: Bool = false,
+            debugDocumentVersioning: Bool = true,
+            debugServiceExtension: String = LaunchAction.defaultDebugServiceExtension,
+            allowLocationSimulation: Bool = true,
+            locationScenarioReference: LocationScenarioReference? = nil,
+            enableGPUFrameCaptureMode: GPUFrameCaptureMode = LaunchAction.defaultGPUFrameCaptureMode,
+            disableGPUValidationMode: Bool = false,
+            enableGPUShaderValidationMode: Bool = false,
+            showGraphicsOverview: Bool = false,
+            logGraphicsOverview: Bool = false,
+            enableAddressSanitizer: Bool = false,
+            enableASanStackUseAfterReturn: Bool = false,
+            enableThreadSanitizer: Bool = false,
+            stopOnEveryThreadSanitizerIssue: Bool = false,
+            enableUBSanitizer: Bool = false,
+            stopOnEveryUBSanitizerIssue: Bool = false,
+            disableMainThreadChecker: Bool = false,
+            disablePerformanceAntipatternChecker: Bool = false,
+            stopOnEveryMainThreadCheckerIssue: Bool = false,
+            additionalOptions: [AdditionalOption] = [],
+            commandlineArguments: CommandLineArguments? = nil,
+            environmentVariables: [EnvironmentVariable]? = nil,
+            language: String? = nil,
+            region: String? = nil,
+            showNonLocalizedStrings: Bool = false,
+            launchAutomaticallySubstyle: String? = nil,
+            storeKitConfigurationFileReference: StoreKitConfigurationFileReference? = nil,
+            customLaunchCommand: String? = nil,
+            customLLDBInitFile: String? = nil
+        ) {
+            self.init(
+                runnable: pathRunnable ?? runnable,
```

Comment:
I also realized that I removed PathRunnable as a parameter in LaunchAction's init method which would also be a breaking change.  I created a convenience init method which adds it back for backwards compatibility. I chose to have it call the designated initializer with it's pathRunnable argument nil coalesced to other runnable argument since it is the more specialized one.

Reply:
Thanks @georgenavarro 

Two small suggestions

* Perhaps we can mark it as deprecated and a message to use `init(runnable: pathRunnable)`
* We can make the convenience `init` have a `pathRunnable` without a default to help the compiler disambiguate (and to flag the deprecation warning)

Reply:
I pushed a commit addressing this feedback. Thanks @kwridan